net/dnscrypt-proxy/Makefile

```diff
@@ -32,6 +32,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/dnscrypt-proxy/Default
   SECTION:=net
   CATEGORY:=Network
+  TITLE:=DNSCrypt-Proxy by Dyne, not actively maintained
```

Comment:
ideally title should be put below submenu.

Reply:
this variable is missing in the wiki, hope I could apply for an account
https://openwrt.org/docs/guide-developer/packages#buildpackage_defines